### ISSUE TITLE
copyurl prevent overwriting with custom name or --auto-filename

### ISSUE THREAD

**Issue by VladKappa** [rclone/rclone#3950]
#### What is your current rclone version (output from `rclone version`)?

```
rclone v1.50.2
- os/arch: linux/amd64
- go version: go1.13.4
```

#### What problem are you are trying to solve?

Add a flag for copyurl to check if there is not an existing file on the destination with the same name to prevent overwriting. Additionally, check if `--auto-filename` will not overwrite an existing file on destination. It was also discussed [here](https://forum.rclone.org/t/copyurl-same-file-name-overwrite/13660).

#### How do you think rclone should be changed to solve that?

Here is the source of the copyurl command: https://github.com/rclone/rclone/blob/master/cmd/copyurl/copyurl.go

Adding the `--no-clobber` flag at the source of the copyurl command it on to operations.CopyURL @ https://github.com/rclone/rclone/blob/master/fs/operations/operations.go
Also, I'd need to implement a destination check to prevent overwriting with the `--auto-filename` .

**Comment by ncw** [comment]
This seems like a reasonable idea - want to have a go?

**Comment by VladKappa** [comment]
I'll work on it and come back with a pull request I guess.

**Comment by ncw** [comment]
I've merged the `--no-clobber` flag (thank you @kfur ) this to master now which means it will be in [the latest beta](https://beta.rclone.org) in 15-30 mins and released in v1.52
